### ISSUE TITLE
[JENKINS-65923] Accessibility issue - Images lacking "alt" text

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-15#747]
<p>Recent UI upgrades caused the build status icons to lose their useful alt text.</p>

<p>Usability for blind people declined therefore.</p>

<p> </p>

<p>See <a href="https://issues.jenkins.io/browse/JENKINS-65850?focusedCommentId=410509&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410509" class="external-link" rel="nofollow">https://issues.jenkins.io/browse/JENKINS-65850?focusedCommentId=410509&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410509</a></p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=maial">maial</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-65923" target="_blank">Accessibility issue - Images lacking "alt" text</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/benebsiny.svg" title="benebsiny's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=benebsiny">benebsiny</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: 2.303-fixed, regression
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2021-08-03T17:38:17+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 3
<li><b>imported</b>: 2025-11-16
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Recent UI upgrades caused the build status icons to lose their useful alt text.</p>

<p>Usability for blind people declined therefore.</p>

<p> </p>

<p>See <a href="https://issues.jenkins.io/browse/JENKINS-65850?focusedCommentId=410509&amp;page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410509" class="external-link" rel="nofollow">https://issues.jenkins.io/browse/JENKINS-65850?focusedCommentId=410509&amp;page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410509</a></p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<i>[Originally related to: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-15/issues?q=in%3Atitle%20JENKINS-26503">JENKINS-26503</a>]</i>

**Comment by jenkins-infra-bot** [comment]
<i>[Originally related to: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-15/issues?q=in%3Atitle%20JENKINS-66174">JENKINS-66174</a>]</i>

**Comment by jenkins-infra-bot** [comment]
<i>[Epic: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-15/issues?q=in%3Atitle%20JENKINS-62268%20label%3Aepic">JENKINS-62268</a>]</i>
